lib/capybara/rack_test/browser.rb

```diff
@@ -30,7 +30,9 @@ def refresh
 
   def submit(method, path, attributes)
     path = request_path if path.nil? || path.empty?
-    process_and_follow_redirects(method, path, attributes, 'HTTP_REFERER' => current_url)
+    uri = build_uri(path)
+    uri.query = '' if method&.to_s&.downcase == 'get'
```

Comment:
not sure if you're doing rubocop / care, but `casecmp('get').zero?` is more optimal here

Reply:
Yeah, but it's also much less readable.  If I created a `get?` method I might go with that as the implementation but not for inline.